### ISSUE TITLE
Sp120 plug does not connect

### ISSUE THREAD

**Issue by jabroersen** [kasteleman/com.innr#17]
I’m Having trouble connecting my sp120 power plug. Homey does not see the device

**Comment by kasteleman** [comment]
Never had any issues pairing the sp120. Did you try a reboot of Homey, wait
a few min and then try again

Op za 13 jul. 2019 om 15:15 schreef jabroersen <notifications@github.com>

**Comment by jabroersen** [comment]
Yup, unplugged it, waited 5 minutes, dit a seperate app reboot also…it just doens’t see the device, while the blue indicator is blinking to show it is in discoverable mode

**Comment by kasteleman** [comment]
If you disable the innr app and try to include the device, does it get
included as a generic device?
What’s the distance between Homey and the plug? Do you have other zigbee
routers/ devices in use?

Op zo 14 jul. 2019 om 16:24 schreef jabroersen <notifications@github.com>

**Comment by jabroersen** [comment]
That didn’t work, but...putting the plug closer to homey while pairing did. Thnx for your assistance!

On 14 Jul 2019, at 16:56, kasteleman <notifications@github.com<mailto:notifications@github.com>> wrote:

If you disable the innr app and try to include the device, does it get
included as a generic device?
What’s the distance between Homey and the plug? Do you have other zigbee
routers/ devices in use?

Op zo 14 jul. 2019 om 16:24 schreef jabroersen <notifications@github.com<mailto:notifications@github.com>>
